docs/devguide/building/index.rst

```diff
@@ -672,6 +672,13 @@ These are all the variables that are exclusive to building OpenDDS with CMake:
   See :ref:`cmake-running-tests` for how to run them.
   The default for this is ``TRUE``.
 
+.. cmake:var:: OPENDDS_COMPILE_WARNINGS
+
+   If not empty, enables compiler warnings when compiling OpenDDS.
```

Comment:
"enables more compiler warnings" ?  Aren't some enabled by default either in CMake, its specific Generators, or the compilers themselves?